plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -532,4 +533,17 @@ public DeltaLakeConfig setDeletionVectorsEnabled(boolean deletionVectorsEnabled)
         this.deletionVectorsEnabled = deletionVectorsEnabled;
         return this;
     }
+
+    public boolean isDeltaLogFileSystemCacheEnabled()
+    {
+        return deltaLogFileSystemCacheEnabled;
+    }
+
+    @Config("delta.fs.cache.delta-log-caching-enable")
+    @ConfigDescription("Enable object storage caching of the _delta_log directory (effective only when fs.cache.enabled=true)")
+    public DeltaLakeConfig setDeltaLogFileSystemCacheEnabled(boolean deltaLogFileSystemCacheEnabled)
+    {
+        this.deltaLogFileSystemCacheEnabled = deltaLogFileSystemCacheEnabled;
+        return this;
+    }
```

Comment:
We could have a new Config class which could be binded if `fs.cache.enabled` is set to true.

Reply:
Right now we know that only in FileSystemModule, pulling that info out in delta connector module would get a little messy. It's not ideal, but I think using existing Config class should also be okay.

Reply:
Since it is only one property, I think for now it could stay in this Config class.